app/serializers/financial_transaction_serializer.rb

```diff
@@ -2,11 +2,16 @@ class FinancialTransactionSerializer < ActiveModel::Serializer
   include ApplicationHelper
 
   attributes :id, :user_id, :user_name, :amount, :note, :created_at
+  attributes :financial_transaction_type_id, :financial_transaction_type_name
 
   def user_name
     show_user object.user
   end
 
+  def financial_transaction_type_name
+    object.financial_transaction_type.name
+  end
+
```

Comment:
I wonder if we should add a nested `financial_transaction_type` object here, or keep it with `id` and `name`?
Indeed account name and IBAN wouldn't really make sense here, so then we'd need separate serialization variants, so perhaps this approach is the best.

Reply:
i had the same thoughts, end ended with the current code :wink: